intel-sgx/ppid-retrieval-tool/pce/Makefile

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) Fortanix, Inc.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+CC		  = gcc
+AR		  = ar
+LD		  = gcc
+EDGER		  = sgx_edger8r
+SIGNER		  = sgx_sign
+INCLUDE           = -I${SGX_SDK}/include                               \
+		    		-I${SGX_SDK}/include/tlibc
+T_CFLAGS	  = $(CFLAGS) -nostdinc -fvisibility=hidden -fpie -fstack-protector -g
+U_CFLAGS	  = $(CFLAGS) -nostdinc -fvisibility=hidden -fpie -fstack-protector -g
+AR_FLAGS	  = rcs
+OBJECTS		  = pce_u.c pce_u.h pce_u.o
```

Comment:
Try to understand:
How are these `*_u` files generated ? Are they generated by sgx_edger8r ?